ampd/src/commands/worker_address.rs

```diff
@@ -1,33 +1,14 @@
-use error_stack::{Report, ResultExt};
 use std::path::Path;
 
-use crate::state::{self, StateUpdater};
-use crate::tofnd::grpc::{MultisigClient, SharableEcdsaClient};
+use error_stack::Result;
+
+use crate::commands::worker_pub_key;
 use crate::tofnd::Config;
```

Comment:
don't import the namespace for the `Config`, otherwise it's too easy to confuse with the root `Config` type